### DISCUSSION TITLE
Tamper Chrome - V3 Manifest Compatibility

### DISCUSSION THREAD

**Discussion by ethanjerrum5** [google/tamperchrome#268] [General]
Good afternoon

I'm contacting you from Airbus Defence and Space as we use the Tamper Chrome Extension within the company. 

Do you have any plans to upgrade this extension to V3 manifest, please? Aligning with Google's plan to deprecate V2 in June 2025

Thank you in advance

Kind Regards,

**Comment by ethanjerrum5** [comment]
Good morning, 

Any update on this, please? 

Kind Regards,
Ethan

**Comment by stephenkall** [comment]
I second on that, Tamper Dev is a very useful tool and considering it might rely on webAPI interception, will it even be posible on MV3?
